llvm/test/CodeGen/AIE/aie2/hardware-loops/noduplication.mir

```diff
@@ -0,0 +1,152 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 4
+# RUN: llc -mtriple=aie2 --start-after=instruction-select \
+# RUN:   --stop-before=aie-finalize-mi-bundles %s -o - | FileCheck %s
```

Comment:
Super-Nit: I'm changing the scheduler so it outputs "correct" Bundles. Could you change that line to `--stop-after=aie-finalize-mi-bundles`? This way there won't be test updates.